src/vsr/replica.zig

```diff
@@ -4506,28 +4666,28 @@ pub fn ReplicaType(
         }
 
         fn send_start_view_change(self: *Self) void {
-            assert(self.status == .view_change);
-            assert(self.log_view < self.view);
-            assert(!self.do_view_change_quorum);
-            // Send only to other replicas (and not to ourself) to avoid a quorum off-by-one error:
-            // This could happen if the replica mistakenly counts its own message in the quorum.
-            self.send_header_to_other_replicas(.{
+            assert(self.status == .normal or self.status == .view_change);
+
+            const header: Header = .{
                 .command = .start_view_change,
                 .cluster = self.cluster,
                 .replica = self.replica,
                 .view = self.view,
-            });
+            };
+
+            self.send_header_to_other_replicas(header);
+
+            if (!self.start_view_change_from_all_replicas.isSet(self.replica)) {
+                self.send_header_to_replica(self.replica, header);
+                defer self.flush_loopback_queue();
+            }
```

Comment:
Why send ourselves an SVC message rather than simply flipping the bit in the quorum count?
This may be the last SVC needed to complete our quorum — using the regular message path means there's only one code path for quorum completion.